.github/workflows/test-win.yml

```diff
@@ -77,9 +85,11 @@ jobs:
         if: env.RUN_BUILD == 'true'
         uses: goanpeca/setup-miniconda@v1
         with:
-           activate-environment: test
-           auto-update-conda: true
-           python-version: ${{ matrix.PYTHON_VERSION }} 
+          activate-environment: test
+          auto-update-conda: true
+          auto-activate-base: false
+          python-version: ${{ matrix.PYTHON_VERSION }}
+          use-only-tar-bz2: true
```

Comment:
It seems this is not a valid conda parameter on Windows?

Reply:
Yes, but only on new versions so the conda needs to have a minimum version. I am working a fix on the action so that this fail does not occur.

Reply:
Ok, thanks!